docs/book/src/faq.md

```diff
@@ -92,9 +92,38 @@ securityContext:
 ```
 However, note that this problem is fixed and will not occur if you deploy the project in high versions (maybe >= 1.22).
 
+## The error `Too long: must have at most 262144 bytes` is faced when I run `make install` to apply the CRD manifests. How to solve it? Why this error is faced?
+
+When attempting to run `make install` to apply the CRD manifests, the error Too long: must have at most 262144 bytes may be encountered. This error arises due to a size limit enforced by the Kubernetes API. Note that the `make install` target will apply the CRD manifest under `config/crd` using `kubectl apply -f -`. Therefore, when the apply command is used, the API annotates the object with the `last-applied-configuration` which contains the entire previous configuration. If this configuration is too large, it will exceed the allowed byte size. ([More info][k8s-obj-creation]) and this error will be faced.
+
+In ideal approach might use client-side apply might seem like the perfect solution since with the entire object configuration doesn't have to be stored as an annotation (last-applied-configuration) on the server. However, it's worth noting that as of now, it isn't supported by controller-gen or kubebuilder. For more on this, refer to: [Controller-tool-discussion][controller-tool-pr].
```

Comment:
```suggestion
The ideal approach is to use server-side apply, since the entire object configuration doesn't have to be stored as an annotation (last-applied-configuration) on the server. However, it's worth noting that as of now, it isn't supported by controller-gen or kubebuilder. For more on this, refer to: [Controller-tool-discussion][controller-tool-pr].
```
@camilamacedo86 @Sajiyah-Salat this should be server-side apply, right?

SSA uses a declarative approach than using the client side apply and storing previous configuration in `last-applied-configuration` field: https://kubernetes.io/docs/reference/using-api/server-side-apply/#field-management. Please correct me if I'm wrong.

Reply:
Yes, @camilamacedo86 talked about the server side in the last [comment](https://github.com/kubernetes-sigs/kubebuilder/pull/3543#discussion_r1300241107). here, I think there is a typo mistake.